app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoQualityController.kt

```diff
@@ -0,0 +1,32 @@
+package org.jellyfin.androidtv.ui.playback
+
+import org.jellyfin.androidtv.preference.UserPreferences
+import org.jellyfin.androidtv.preference.UserPreferences.Companion.maxBitrate
+import org.koin.java.KoinJavaComponent.get
+import org.jellyfin.androidtv.constant.QualityProfiles
+
+class VideoQualityController(
+	@Suppress("unused")
+	private val parentController: PlaybackController
+) {
+	companion object {
+		private var previousQualitySelection = QualityProfiles.fromPreference(
+				get<UserPreferences>(UserPreferences::class.java)
+						.get(maxBitrate))
+	}
+
+	var currentQuality = previousQualitySelection
+		@Suppress("unused")
```

Comment:
Why suppress a unused fucntion instead of removing it?

Reply:
https://github.com/jellyfin/jellyfin-androidtv/pull/1924/files#diff-58f252ef3b690abba89c1a33e1c5bafe249c3e1dcb26aad7ad76795b9537fd07:~:text=menuItem.itemId%5D.quality)-,qualityController.currentQuality,-%3D%20QualityProfiles.fromPreference